R/derive_vars_query.R

```diff
@@ -58,8 +61,8 @@
 #'   "05", "2020-06-09 23:59:59", "ALVEOLAR PROTEINOSIS",
 #'   7, "Alveolar proteinosis", NA_character_, NA_integer_
 #' )
-#' derive_vars_query(adae, queries)
-derive_vars_query <- function(dataset, dataset_queries) { # nolint: cyclocomp_linter
+#' derive_vars_for_query(adae, queries)
+derive_vars_for_query <- function(dataset, dataset_queries) { # nolint: cyclocomp_linter
```

Comment:
The function needs to be renamed because it removes variables from the input dataset. I would suggest naming it `get_vars_query()`.